agent/agent.go

```diff
@@ -266,7 +266,6 @@ func (a *agent) createTailnet(ctx context.Context, derpMap *tailcfg.DERPMap) (ne
 			network.Close()
 		}
 	}()
-	a.network = network
```

Comment:
Let's be mindful about this one if we notice anything weird with the stats agent. Some stats message may be discarded until `a.network` is reassigned. But maybe this won't matter?

Reply:
Yes, I will keep an eye on this one.